packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -221,66 +221,73 @@ async function getOrderNumberFromSession(
 // Returns the regionalized orderForm
 const getOrderForm = async (
   id: string,
-  session: Maybe<IStoreSession> | undefined,
   { clients: { commerce } }: Context
 ) => {
   const orderForm = await commerce.checkout.orderForm({
     id,
   })
 
+  return orderForm
```

Comment:
Return orderform directly or remove this function and move this call to the validateCart function directly.